src/styles/variables.css

```diff
@@ -0,0 +1,15 @@
+:root {
+  --card-max-width: calc(200px - 2px);
```

Comment:
Check this property: 
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing
With box-sizing: border-box there will be no need to calculate the size